tools/topology/sof/pipe-eq-capture.m4

```diff
@@ -81,7 +80,8 @@ W_PCM_CAPTURE(PCM_ID, Highpass Capture, 0, 2)
 # "Volume" has 2 source and 2 sink periods
 W_PGA(0, PIPELINE_FORMAT, 2, 2,
 	 capture_pga_conf, LIST(`		',
-		"CONTROL_NAME_VOLUME", "CONTROL_NAME_SWITCH"))
+		ifdef(`CONTROL_NAME_VOLUME', `"CONTROL_NAME_VOLUME"', `"PIPELINE_ID Master Capture Volume"'),
+		ifdef(`CONTROL_NAME_SWITCH', `"CONTROL_NAME_SWITCH"', `"PIPELINE_ID Master Capture Switch"')))
```

Comment:
@juimonen and this seems incorrect to me. If you want the names to be passed from the top-level, I think we should check if they are defined and if not throw an error, right?

Reply:
@ranj063 the idea was that if the names are not defined we use the "old" legacy method of naming the pga's.